src/client/localplayer.cpp

```diff
@@ -55,9 +55,12 @@ static aabb3f getNodeBoundingBox(const std::vector<aabb3f> &nodeboxes)
 	return b_max;
 }
 
+
 bool LocalPlayer::updateSneakNode(Map *map, const v3f &position,
 	const v3f &sneak_max)
 {
+	// Acceptable distance to node center
+	constexpr f32 allowed_range = (0.5f + 0.1f) * BS;
```

Comment:
This seems pretty hardcoded and arbitrary to me

Reply:
I could've as well changed nothing and nobody would ever complain.

It's good as is and should be kept like that.

Reply:
I would at least annotate this as a hack. The "allowed range" should be based on nodebox extent.

Reply:
setting it to nodebox extent = no range checking = the bug seen in OP issue